frontend/src/components/auth/Credentials.tsx

```diff
@@ -41,20 +42,29 @@ const Credentials = ({
   const history = useHistory();
 
   const onLogInClick = async () => {
-    const loginResponse: LoginResponse = await authAPIClient.login(
-      email,
-      password,
-    );
-    if (loginResponse) {
-      const { requiresTwoFa, authUser } = loginResponse;
-      if (requiresTwoFa) {
-        setToggle(!toggle);
-      } else {
-        localStorage.setItem(AUTHENTICATED_USER_KEY, JSON.stringify(authUser));
-        setAuthenticatedUser(authUser);
+    const isInvited = await commonApiClient.isUserInvited(email);
+    if (isInvited) {
+      const loginResponse: LoginResponse = await authAPIClient.login(
+        email,
+        password,
+      );
+      if (loginResponse) {
+        const { requiresTwoFa, authUser } = loginResponse;
+        if (requiresTwoFa) {
+          setToggle(!toggle);
+        } else {
+          localStorage.setItem(
+            AUTHENTICATED_USER_KEY,
+            JSON.stringify(authUser),
+          );
+          setAuthenticatedUser(authUser);
+        }
       }
+      // Otherwise we can display some sort of error
+    } else {
+      // eslint-disable-next-line no-alert
+      window.alert("user not invited");
```

Comment:
we also need to add a catch for when the user already exists on the app

Reply:
later ticket? i'll add as a todo, also think this would only be on the signup flow not the login